helm-chart/renku-notebooks/templates/configmap.yaml

```diff
@@ -23,3 +23,31 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   SENTRY_DSN: {{ .Values.jupyterhub.singleuser.sentryDsn }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hub-helper-scripts
```

Comment:
this is a very minor thing, but technically this is a helper script for jupyter notebooks not the hub... maybe the naming will be misleading?

Reply:
`notebook-helper-scripts` then?

Reply:
👍